src/Sarif.FunctionalTests/v2/ConverterTestData/FortifyFpr/FortifyTest.fpr.sarif

```diff
@@ -47,10 +47,10 @@
                   "uri": "file:///C:/repo/src/ContentFileViewModel.cs"
                 },
                 "region": {
-                  "startLine": 3,
-                  "endLine": 9,
+                  "startLine": 6,
+                  "endLine": 6,
                   "snippet": {
-                    "text": "\nnamespace MS.IT.HRE.CMOE.Web.Models\n{\n    public class ContentFileViewModel : BaseViewModel\n    {\n        #region Public Properties\n\n"
+                    "text": "    public class ContentFileViewModel : BaseViewModel"
```

Comment:
>public [](start = 33, length = 6)

Should we be populating the 'contextRegion' property as well? i.e., does Fortify persist a larger surrounding snippet?

Reply:
Yes! I knew there was a place for this. Done.

---
In reply to: [210471704](https://github.com/Microsoft/sarif-sdk/pull/1020#discussion_r210471704) [](ancestors = 210471704)